test/e2e/helpers.js

```diff
@@ -460,6 +460,20 @@ const onboardingCompleteWalletCreation = async (driver) => {
   await driver.clickElement('[data-testid="onboarding-complete-done"]');
 };
 
+const onboardingCompleteWalletCreationWithOptout = async (driver) => {
+  // wait for h2 to appear
+  await driver.findElement({ text: 'Wallet creation successful', tag: 'h2' });
+  // opt-out from third party API
+  await driver.clickElement({ text: 'Advanced configuration', tag: 'a' });
+  await Promise.all(
+    (
+      await driver.findClickableElements('.toggle-button.toggle-button--on')
```

Comment:
Interesting, so this activating _all_ toggles? Offhand I'm not even sure that they all start in the "on" position (e.g. if the goal here is to disable everything, this might be enabling certain things)

Reply:
I remember that while manually testing the toggles all showed up as activated, but to be extra sure I added `.toggle-button--on` to the query selector - this way we'll only click on the enabled ones

Reply:
Ahh I missed the `--on`, makes sense